provisioning_templates/provision/kickstart_default.erb

```diff
@@ -50,7 +50,10 @@ skipx
 <% else -%>
 <% dhcp = !@static -%>
 <% end -%>
-network --bootproto <%= dhcp ? 'dhcp' : "static --ip=#{@host.ip} --netmask=#{subnet.mask} --gateway=#{subnet.gateway} --nameserver=#{[subnet.dns_primary, subnet.dns_secondary].select{ |item| item.present? }.join(',')}" %> --hostname <%= @host %><%= os_major >= 6 ? " --device=#{@host.mac}" : '' -%>
+<% if subnet.respond_to?(:mtu) && !subnet.mtu.blank? -%>
+<% mtu = subnet.mtu.to_s %>
```

Comment:
This makes mtu empty string in case it's nil, in line below it adds mtu param `if mtu` which would be true in this case, since empty string are evaluated as true. The same applies for user leaving the text field empty. I wonder if it wouldn't be better to always have mtu value set on subnet as required field and default to 1500.

Reply:
Actually it would not be an issue since there's check for blank on line above, but still, wouldn't it be better if we couls rely on mtu would always be there?